README.md

```diff
@@ -21,7 +21,10 @@ Released under [Apache 2.0](/LICENSE) by [@boostsecurityio](https://github.com/b
 
 ---
 
-**Prior art**
-
-This project is largely inspired from previous projects such as https://gtfobins.github.io/ and https://lolbas-project.github.io/.
+# Prior art / Credits
 
+This project is largely inspired from previous projects such as:
+- https://gtfobins.github.io
+- https://lolbas-project.github.io
+- https://github.com/rotem-cider/cicd-lamb
+- https://alex.kaskaso.li/post/terraform-plan-rce
```

Comment:
why is this credited in the readme? it would be more useful in the entry for terraform.